app/context/DeepgramContextProvider.js

```diff
@@ -23,7 +23,7 @@ const DeepgramContextProvider = ({ children }) => {
 
     setSocketState(0); // connecting
 
-    const newSocket = new WebSocket("wss://sts.sandbox.deepgram.com/agent", [
+    const newSocket = new WebSocket("wss://agent.deepgram.com/agent", [
```

Comment:
@DamienDeepgram is this correct? DX have been told it will be was://api.deepgram.com/v1/agent 

cc @jpvajda